src/test/java/TestClass.java

```diff
@@ -6,48 +6,56 @@
 import org.openqa.selenium.Point;
 
 import java.time.Duration;
-import java.util.Random;
 
 import static com.codeborne.selenide.Selenide.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class TestClass extends BaseTest {
-    public static String baseUrl = "http://automationpractice.com/";
+    static final String BASE_URL = "http://automationpractice.com/";
```

Comment:
find a better place in code for this variable

Reply:
moved BASE_URL to the right place